<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSetToolOffsetDistance</class>
 <widget class="QDialog" name="DialogSetToolOffsetDistance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>970</width>
    <height>830</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_image_bg">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>291</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>NAME : </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_name">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_name_change">
      <property name="text">
       <string>Change</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_image">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>291</width>
     <height>19</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>ID : </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_id">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_get_base_image">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>550</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Get Base Image</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>60</y>
     <width>291</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>Object</string>
   </property>
   <widget class="QPushButton" name="pushButton_select_object">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Object</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_reset_object">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>200</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset Object</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>251</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Offset Direction</string>
    </property>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>↓</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>↑</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>→</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>←</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
